docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -365,12 +365,13 @@ const Clicker = ({ click }) => (
 The test finishes before the component calls the `click` prop twice, and without
 retrying the assertion `expect(onClick).to.be.calledTwice`.
 
-It could also fail if React decides to rerender the DOM between clicks.
+It could also fail if React or another javascript library decided to rerender
+the DOM between clicks.
 
 #### <Icon name="check-circle" color="green" /> Correctly waiting for the stub to be called
 
 We recommend aliasing the stub using the [`.as`](/api/commands/as) command and
-using `cy.get('@alias').should(...)` assertions.
+using `cy.get('@alias')` to run assertions.
```

Comment:
```suggestion
using `cy.get('@alias')` to re-run a set of queries and assertions.
```

Reply:
This change doesn't make much sense to me - we're giving advice directly on how to run assertions about stubs.